snuba/settings/settings_test_distributed_migrations.py

```diff
@@ -26,7 +26,7 @@
         "storage_sets": {
             "migrations",
         },
-        "single_node": False,
+        "single_node": True,
```

Comment:
I don't think you need to revert this part

Reply:
We can add it back in later, but for now we need to get master unblocked.